pyproject.toml

```diff
@@ -25,6 +25,7 @@ requests = "*"
 importlib-metadata = "^3.7.3"
 svix = "^0.41.2"
 typed-ast="^1.5.4"
+pydantic = "^1.10.7"
```

Comment:
Hmmm... interesting choice, imho we dont need the validation on the client as well since the server already does that. the documentation is enough to ensure the schemas are correct. Otherwise we introduce a large maintenance burden. We can do TypedDict instead and at least that is not checked at runtime and we can make backwards compatible changes to the server without breaking old client code

Reply:
Though I am curious if pydantic is a popular dependency of similar python API wrappers.